### PR TITLE
Add a fetch request to fill out a project section and to add a weathe…

### PR DESCRIPTION
**Pull request by alenamedved**
…r to the page using geolocation. Add a bar menu to mobile version. Add a photo scroll to Winnie's page.  Some changes to the css files.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -7,51 +7,51 @@ document.addEventListener('DOMContentLoaded', () => {
...
     arrImg[1] = new Image
     arrImg[0].src = "img/star_logo.png"
     arrImg[1].src = "img/heart_logo.jpg"
     arrImg[0].width = 16
     arrImg[1].width = 16
-    
+
+    //check if radio button is checked or not
     function checkRadioButton() {
         let radio = document.getElementsByName('radio')
```
Comment (ginestah) [inline js/index.js:23]: just use const if not reassigning. Doesn't matter that much, just better practice.

File: js/index.js [hunk 2]
```diff
@@ -100,10 +100,10 @@ document.addEventListener('DOMContentLoaded', () => {
     });
 
     skillsSection.addEventListener('mouseout', (event) => {
-        if(event.target.tagName == 'LI') {
-            for(let i = 0; i < skills.length; i++) {
-                if(event.target.textContent == skills[i].toUpperCase()) {
-                event.target.textContent = skills[i]
+        if (event.target.tagName == 'LI') {
+            for (let i = 0; i < skills.length; i++) {
+                if (event.target.textContent == skills[i].toUpperCase()) {
```
Comment (ginestah) [inline js/index.js:105]: The toUpperCase is a little jarring, and messes with the ui quite a bit, maybe if you add a fixed width on the skills this wont happen. It doesn't matter that much, just something I noticed.

File: js/index.js [hunk 3]
```diff
@@ -114,35 +114,35 @@ document.addEventListener('DOMContentLoaded', () => {
...
-        
+
         //Collect an input data from the user and store it in variables name, email, messageFromUser
         const name = event.target.name.value
         const email = event.target.email.value
         const messageFromUser = event.target.messageFromUser.value
-        
+
         //Log to the console to check if it works
         console.log(`${name} ${email} wrote: ${messageFromUser}`)
```
Comment (ginestah) [inline js/index.js:124]: remove console.logs now that you are done.

File: js/location.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+//if user blocks the location the browser will show the weather in Arvada, CO
+//post weather pin by location
+if(window.screen.width >= 800) {
+navigator.geolocation.getCurrentPosition(function (position) {
+    latitude = position.coords.latitude.toFixed(2)
+    longitude = position.coords.longitude.toFixed(2)
+    
+    console.log(`latitude: ${latitude}`)
```
Comment (ginestah) [inline js/location.js:8]: remove console.logs if you are done.

**Review by ginestah — APPROVED** [review]
Really great job with everything, you are really talented and you always go above and beyond!
Just remember to remove commented out code unless it's describing functionality that isn't obvious. And const for variables if not reassigning.

Great job with media queries, looks great on multiple sizes. Well done!

File: js/index.js (hunk 2 above)
Comment (alenamedved) [reply js/index.js:105]: thanks for the opinion, didn't think about that this way

File: js/location.js (hunk 4 above)
Comment (alenamedved) [reply js/location.js:8]: Thank you very much for your review. Really appreciate it. Will follow all your advices

**Pull request merged**